apps/dev-tools/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "dev-tools",
+	"private": true,
+	"description": "tldraw dev tools",
+	"version": "2.0.0-beta.2",
+	"type": "module",
+	"scripts": {
+		"dev": "vite",
+		"build": "tsc -b && vite build",
+		"lint": "yarn run -T tsx ../../scripts/lint.ts",
+		"preview": "vite preview"
+	},
+	"dependencies": {
```

Comment:
our other deps have:
```
"peerDependencies": {
		"react": "^18",
		"react-dom": "^18"
	},
```

let's make them match - i don't want to maybe have different versions of react

Reply:
Using just 18 gives me some yarn constraints issues
![CleanShot 2024-09-12 at 12 16 26](https://github.com/user-attachments/assets/205c8064-a235-4e47-9b3b-c09e21c2575e)

We also seem to be using both 18.2 and just 18 all over the place. Will keep it as is for now.
![image](https://github.com/user-attachments/assets/97916d76-1f29-4c97-8bf0-d5fe837c3ca6)